lib/internal/modules/esm/get_format.js

```diff
@@ -71,9 +74,18 @@ function defaultGetFormat(url, context, defaultGetFormatUnused) {
         throw new ERR_UNKNOWN_FILE_EXTENSION(ext, fileURLToPath(url));
       }
     }
-    return { format: format || null };
-  }
-  return { format: null };
+
+    return format || null;
+  },
+  'node:'() { return 'builtin'; },
+});
+
+function defaultGetFormat(url, context) {
```

Comment:
I re-named it to `defaultGetFormat` to make it more clear in the stack-trace, but I'm not married to it.

Reply:
I think `default*` names make sense for the hooks that users can override (so `defaultResolve`, `defaultLoad`) but not otherwise; `defaultGetFormat` implies that there should be a hook `getFormat` that users can define to override `defaultGetFormat`.

Reply:
Okay. I still don’t like the name `defaultGetFormat` but we don’t need to fix it in this PR.

Reply:
Mm, I agree outside ESMLoader it doesn't make much sense. I thought perhaps `internalGetFormat`, but meh.

I'm also a bit wary to change it because it will break a lot of message tests, which are a pain to find and update 😅

I'd be happy to address in a follow-up 🙂